src/RNGestureHandlerModule.web.ts

```diff
@@ -39,16 +41,20 @@ export default {
     NodeManager.createGestureHandler(handlerTag, new GestureClass());
     this.updateGestureHandler(handlerTag, config);
   },
-  attachGestureHandler(handlerTag, newView, propsRef) {
+  attachGestureHandler(
+    handlerTag: number,
+    newView: number,
+    propsRef: React.RefObject<unknown>
+  ) {
     NodeManager.getHandler(handlerTag).setView(newView, propsRef);
   },
-  updateGestureHandler(handlerTag, newConfig) {
+  updateGestureHandler(handlerTag: number, newConfig: any) {
```

Comment:
Will it work?
```suggestion
  updateGestureHandler(handlerTag: number, newConfig: object) {
```

Reply:
Yeah, but config is more constrained than `object`, `Record<string, unknown>` would be better but let's keep `any` until we'll be sure which properties can go into `newConfig` and which cannot.